src/util/HeaderUtil.ts

```diff
@@ -108,6 +108,11 @@ export interface ContentType {
   parameters: Record<string, string>;
 }
 
+export interface LinkEntry {
+  link: string;
+  params: Record<string, string>;
```

Comment:
```suggestion
  parameters: Record<string, string>;
```
(for consistency, I think)